src/index.js

```diff
@@ -6,20 +6,23 @@ import { makeServer } from "./server";
 import { FilterProvider } from "./context/filter-context";
 import { WishlistProvider } from "./context/wishlist-context";
 import { CartProvider } from "./context/cart-context";
+import { AuthProvider } from "./context/auth-context";
```

Comment:
Can import all these in a single line from `./context` since you have added `index` file in context